data/table_coordinates.py

```diff
@@ -0,0 +1,115 @@
+def get_classic_coordinates() -> dict[str, tuple[float, float]]:
```

Comment:
could you write a bit what these are in the docstring? I am not familiar with these element coordinates (like 14, 1.5) and the special coordinates. You can use the numpy docsring just """ """

and for more of course https://numpydoc.readthedocs.io/en/latest/format.html